src/comment.rs

```diff
@@ -484,7 +484,9 @@ impl ItemizedBlock {
         // allowed.
         for suffix in [". ", ") "] {
             if let Some((prefix, _)) = trimmed.split_once(suffix) {
-                if prefix.len() <= 2 && prefix.chars().all(|c| char::is_ascii_digit(&c)) {
+                if (1..=2).contains(&prefix.len())
```

Comment:
Oh I see. If the `prefix.len() == 0` the condition would still pass. Good catch. I'm wondering if there's a more obvious way to write this since it took me a few seconds to figure out what `(1..=2).contains(..)` was doing.

Maybe assigning the condition to a meaningful variable name would help.

```rust
let has_leading_digits = (1..=2).contains(&prefix.len()) && prefix.chars().all(|c| char::is_ascii_digit(&c));
```